models/curvefi/fantom/curvefi_fantom_trades.sql

```diff
@@ -22,17 +23,18 @@ WITH exchange_evt_all as (
     {%- for src in evt_TokenExchange_sources %}
     SELECT
         evt_block_time AS block_time,
+        evt_block_number as block_number,
         buyer AS taker,
         tokens_bought AS token_bought_amount_raw,
         tokens_sold AS token_sold_amount_raw,
-        bought_id,
-        sold_id,
+        COALESCE(bought_id_int256, CAST(bought_id_uint256 as INT256)) as bought_id,
+        COALESCE(sold_id_int256, CAST(sold_id_uint256 as INT256)) as sold_id,
```

Comment:
related height ticket, as to why we have multiple columns for same field with different data types:
https://dune.height.app/T-31048